contracts/snowglobes/traderJoe/snowglobe-joe-avax-fief.sol

```diff
@@ -0,0 +1,136 @@
+// https://github.com/iearn-finance/vaults/blob/master/contracts/vaults/yVault.sol
+
+pragma solidity ^0.6.7;
+
+import "../../interfaces/controller.sol";
+
+import "../../lib/erc20.sol";
+import "../../lib/safe-math.sol";
+
+contract SnowGlobeJoeAvaxFief is ERC20 {
+    using SafeERC20 for IERC20;
+    using Address for address;
+    using SafeMath for uint256;
+
+    IERC20 public token;
+
+    uint256 public min = 9500;
+    uint256 public constant max = 10000;
+
+    address public governance;
+    address public timelock;
+    address public controller;
+
+    constructor(
+        address _token,
+        address _governance,
+        address _timelock,
+        address _controller
+    )
+        public
+        ERC20(
+            string(abi.encodePacked("freezing ", ERC20(_token).name())),
+            string(abi.encodePacked("s", ERC20(_token).symbol()))
+        )
+    {
+        _setupDecimals(ERC20(_token).decimals());
+        token = IERC20(_token);
+        governance = _governance;
+        timelock = _timelock;
+        controller = _controller;
+    }
+
+    function balance() public view returns (uint256) {
+        return
+            token.balanceOf(address(this)).add(
+                IController(controller).balanceOf(address(token))
+            );
+    }
+
+    function setMin(uint256 _min) external {
+        require(msg.sender == governance, "!governance");
+        require(_min <= max, "numerator cannot be greater than denominator");
+        min = _min;
+    }
+
+    function setGovernance(address _governance) public {
+        require(msg.sender == governance, "!governance");
+        governance = _governance;
+    }
+
+    function setTimelock(address _timelock) public {
+        require(msg.sender == timelock, "!timelock");
+        timelock = _timelock;
+    }
+
+    function setController(address _controller) public {
+        require(msg.sender == timelock, "!timelock");
+        controller = _controller;
+    }
+
+    // Custom logic in here for how much the globes allows to be borrowed
+    // Sets minimum required on-hand to keep small withdrawals cheap
```

Comment:
This comment is slightly misleading as this function doesn't set anything.

Reply:
Removed from snowglobe